### PR TITLE
Navbar for internal site navigation & homepage revamp

### PR DESCRIPTION
**Pull request by swarajpure**
image
image
image
image
image

### CODE REVIEW

**Comment by swarajpure** [comment]
image

On Hover:

image

**Comment by swarajpure** [comment]
New screenshots are updated in the PR description

**Review by prakashchoudhary07 — DISMISSED** [review]

File: app/components/local-navbar.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<nav class="local-navbar">
+  <p class="local-navbar__link">
+    <LinkTo @route="index">Home</LinkTo>
+  </p>
```
Comment (ankushdharkar) [inline app/components/local-navbar.hbs:4]: Should this be a <span> like the rest?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

**Comment by swarajpure** [comment]
- Remove `signup`
- Let the navbar elements come from a list of objects

**Pull request merged**